pkg/meta/model/job_args.go

```diff
@@ -41,6 +43,146 @@ type JobArgs interface {
 	fillJob(job *Job)
 }
 
+// FinishedJobArgs is the interface for finished job arguments.
+// in most cases, job args are cleared out after the job is finished, but some jobs
+// will write some args back to the job for other components.
+type FinishedJobArgs interface {
```

Comment:
```suggestion
type finishedJobArgs interface {
```

Reply:
it's used in some test, see https://github.com/pingcap/tidb/pull/55919/files#diff-90a20211c2de727da7d84a8bb29745bf631f43fa1a30394e4f60adfadcd7276cR727